Library/Homebrew/attestation.rb

```diff
@@ -0,0 +1,89 @@
+# typed: true
+# frozen_string_literal: true
+
+require "date"
+require "json"
+require "utils/popen"
+require "exceptions"
+
+module Homebrew
+  module Attestation
+    # @api private
+    HOMEBREW_CORE_REPO = "Homebrew/homebrew-core"
+    # @api private
+    HOMEBREW_CORE_CI_URI = "https://github.com/Homebrew/homebrew-core/.github/workflows/publish-commit-bottles.yml@refs/heads/master"
+
+    # @api private
+    BACKFILL_REPO = "trailofbits/homebrew-brew-verify"
+    # @api private
+    BACKFILL_REPO_CI_URI = "https://github.com/trailofbits/homebrew-brew-verify/.github/workflows/backfill_signatures.yml@refs/heads/main"
+
+    # No backfill attestations after this date are considered valid.
```

Comment:
Might be nice to have a non-here source linked to as to why this date is what it is